test/files/README.md

```diff
@@ -0,0 +1,12 @@
+## e2e test suite
```

Comment:
nice, maybe link this to the e2e test in case people discover this from the test code angle